test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/main.py

```diff
@@ -501,13 +510,15 @@ def _find_module_utils(self, main):
                             if self._is_new_module():
                                 self.reporter.error(
                                     path=self.object_path,
+                                    name='FIXME',
```

Comment:
I assume this and the other `FIXME` names are placeholders for names not yet decided?

Reply:
Yes.